prow/cluster/prow-controller-manager_rbac.yaml

```diff
@@ -111,3 +111,13 @@ subjects:
 - kind: ServiceAccount
   name: "prow-controller-manager"
   namespace: default
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
I think this will create service account in Prow build cluster whereas the job actually runs in 'trusted' Prow workers cluster?
If I understand correctly, we don't have any automation at the moment to create Kubernetes resources in the trusted cluster

Reply:
Removed.